### PR TITLE
Stojanovic/ep 49 localvocabularyfield to keep the entire vocabulary record

### PR DESCRIPTION
**Pull request by Ducica**
Added hierarchical display (breadcrumbs) to LocalVocabularyInput for hierarchical vocabularies. Note: necessary to merge also https://github.com/Narodni-repozitar/nr-docs/pull/64 and https://github.com/oarepo/oarepo-ui/pull/108

Note: we still have issue with the fact that we are always displaying all of items and not just end nodes how it is supposed to be in eg. Licenses input.

### CODE REVIEW

**Comment by Ducica** [comment]
@mirekys Zdar, I think it should be Ok now please take a look. 

Notes: 

1. necessary to merge (checkout) also https://github.com/Narodni-repozitar/nr-docs/pull/64 and https://github.com/oarepo/oarepo-ui/pull/108
2. Translations are probably missing, as we are transitioning now, I thought best to wait until we merge everything and then use the new principle to extract all untranslated strings and translate them
3. I included datepicker javascript (less) in javascript.html in oarepo ui as DepositForm (nr-docs) template is currently not inheriting from form template in oarepo ui

**Review by mirekys — APPROVED** [review]

**Pull request merged**
